GlideDateTime/Get Next Monday Date/GetNextMondayDate.js

```diff
@@ -1,12 +1,15 @@
-function GetNextMonday() {
-    var date = new GlideDateTime();
-    var today = new GlideDate().getDayOfMonthNoTZ(); //Day number of the month
-    var dayInWeek = date.getDayOfWeek();
-    
-    var nextMonday = (today - dayInWeek + 8); //8 for Monday of Next week. 
-    date.addDaysLocalTime(nextMonday); // interger of days until next monday
-    gs.info("The next upcoming Monday is on " + date.getDate() + ", in "+ nextMonday + " days.");
-    return date.getDate();
+function getNextMonday() {
+    var today = new GlideDate().getDayOfMonthNoTZ();
```

Comment:
this can be removed since it isn't used anymore. otherwise it looks good :)

Reply:
Thanks for your feedback. I will look into it.

Reply:
If you remove line 2 and commit and push the code it will automatically update this pull request. Give it a try 😄 

I see the preview here shows more than line 2 so maybe I'll clarify that `var today` was defined but not used elsewhere in the function so my comment was just that line 2 isn't needed.

I think I've said line 2 enough times. 😅

Reply:
Thank you for the help. I rewrote the code and removed that part. :)

Reply:
@Avinash238 any update?

Reply:
Yes it got merged after the update. Thanks for your guidance since I am trying to get it merged. Could we connect on some platform for assistance?

Reply:
most of us are on Slack (sndevs.com) in the #hacktoberfest channel :)